packages/vscode-extension/package.json

```diff
@@ -927,6 +931,23 @@
                   }
                 },
                 "additionalProperties": false
+              },
+              {
+                "type": "object",
+                "required": [
+                  "configFile"
+                ],
+                "properties": {
+                  "configFile": {
+                    "type": "string",
+                    "default": "${teamsfx:workspaceFolder}/.fx/configs/ngrok.yml"
+                  },
+                  "binFolder": {
+                    "type": "string",
+                    "default": "${teamsfx:ngrokBinFolder}"
```

Comment:
What `default` means? If `binFolder` is empty in `task.json`, we will resolve it as `${teamsfx:ngrokBinFolder}`?

Reply:
No, the default is only used in the auto prompt of vscode.
If the binFolder is empty in the `task.json`, the global ngrok will be used.

Reply:
Then I think we should remove this `default` since the default means the actual behavior when the property is not provided in `tasks.json`.

Are the following mappings what we expect?
- `binFolder: "${teamsfx:ngrokBinFolder}"` -> resolve ngrok in ~/.fx
- `binFolder: "xxx"` -> xxx
- `binFolder: ""` -> resolve global ngrok
- not provided -> resolve global ngrok

Reply:
> Then I think we should remove this `default` since the default means the actual behavior when the property is not provided in `tasks.json`.
> 
> Are the following mappings what we expect?
> 
> * `binFolder: "${teamsfx:ngrokBinFolder}"` -> resolve ngrok in ~/.fx
> * `binFolder: "xxx"` -> xxx
> * `binFolder: ""` -> resolve global ngrok
> * not provided -> resolve global ngrok

Yes, the mapping is the expect. I will use another way to show the auto prompt .

Reply:
Removed the default value.